e2e/articles.spec.ts

```diff
@@ -220,4 +220,58 @@ test.describe("Authenticated Articles Page", () => {
 
     await expect(page.getByText(commentContent)).toBeVisible();
   });
+
+  test("Should sort articles by Newest", async ({ page }) => {
```

Comment:
Do we want to copy and paste these tests into the unauthenticated view?

Does it need to be in both or just unauthenticated?

Reply:
hm... Yes, I guess it shouldnt matter really?
I will copy it to unauthenticated.